booking-module/k6script.js

```diff
@@ -0,0 +1,14 @@
+import http from 'k6/http';
+
+const ids = [];
+
+for(let i = 0; i < 500000; i += 1) {
+  ids.push(Math.floor(Math.random() * 10000000));
+}
+
```

Comment:
Check minimum bound on your random result. As written you could end up with a 0.